languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/custom/grammar.xml

```diff
@@ -164,4 +164,18 @@ USA
             </rule>
         </rulegroup>
     </category>
+    <category id="CONFUSED_WORDS" name="Commonly Confused Words" type="misspelling">
+        <rulegroup id="AFFECT_EFFECT" name="affect vs effect">
+            <rule>
+                <pattern>
+                    <token>affects</token>
+                </pattern>
+                <message>Did you mean <suggestion>effects</suggestion>
+                </message>
+                <short>Commonly confused word</short>
+                <example correction="effects">Children need to understand the<marker>effects</marker> of their actions on others.
+                </example>
+            </rule>
+        </rulegroup>
+    </category>
```

Comment:
We won't be able to use this rule. It is too broad. The rule that it is adapted from makes sense since "side affects" is a more narrow pattern to match. Matching on just "affects" will result in too many false positives as it will match even when "affects" is used correctly.